StellarAssetSdk/Runner.cs

```diff
@@ -58,6 +61,13 @@ public static async Task Run(Server server, KeyPair sgIssuer, KeyPair sgOperator
         claimBalanceTx = await assetIssuer.ClaimClaimableBalanceTransaction(sgOperatorAccount, responses.Records[0].Id.HexToByteArray());
         claimBalanceTx.Sign(sgOperator);
         res = await server.SubmitTransaction(claimBalanceTx);
+
+        var eurcvBalanceAfter = await server.GetAccountBalanceForAsset(assetIssuer.Eurcv,sgOperatorAccount.AccountId);
+        Console.WriteLine(eurcvBalanceAfter);
+
+        if (eurcvBalanceAfter == eurcvBalanceBefore) {
```

Comment:
We probably want to check that it's the amount we expected it be different, but this is good enough for now.